README.md

```diff
@@ -7,6 +7,38 @@ This collection of templates is related to models and automating their conversio
 ## Model
 Automatically generates an initializer and an enum for MySQL enum types.
 
+Example:
+```swift
+final class User: Model {
```

Comment:
Missing annotation with `model` this should be introduced somewhere as well. Sourcery will only pick up types with the `model` annotation.